setup/env.sh

```diff
@@ -243,22 +261,33 @@ then
   export LLMDBENCH_CONTROL_DEPENDENCIES_CHECKED=1
 fi
 
-if [[ $LLMDBENCH_CONTROL_CLI_OPTS_PROCESSED -eq 0 ]]; then
-  return 0
-fi
+function get_latest_image {
```

Comment:
This only gets the latest image when `image_tag` is `auto`, so the function name should describe the more general case.
```suggestion
function get_image {
```